registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/utilities/RegistrationExternalStatusUtility.java

```diff
@@ -64,11 +64,12 @@ public RegistrationExternalStatusCode getExternalStatus(RegistrationStatusEntity
 		} else if (status.equalsIgnoreCase(RegistrationStatusCode.PROCESSING.toString())
 				|| status.equalsIgnoreCase(RegistrationStatusCode.PAUSED.toString())
 				|| status.equalsIgnoreCase(RegistrationStatusCode.RESUMABLE.toString())
-				|| status.equalsIgnoreCase(RegistrationStatusCode.REPROCESS.toString())
-				|| status.equalsIgnoreCase(RegistrationStatusCode.PAUSED_FOR_ADDITIONAL_INFO.toString())) {
+				|| status.equalsIgnoreCase(RegistrationStatusCode.REPROCESS.toString())) {
 			mappedValue = checkStatusforPacketReceiver(entity);
 		} else if (status.equalsIgnoreCase(RegistrationStatusCode.FAILED.toString())) {
 			mappedValue = checkStatusforPacketUploader(entity);
+		} else if (status.equalsIgnoreCase(RegistrationStatusCode.PAUSED_FOR_ADDITIONAL_INFO.toString())) {
+			mappedValue = RegistrationExternalStatusCode.AWAITING_INFORMATION;

```

Comment:
If we are following a separate enum and logic, these changes are not required

Reply:
fixed